setup.py

```diff
@@ -110,6 +110,7 @@ def _read_file(filename):
       include_package_data=True,
       zip_safe=False,
       ext_modules=ext_modules,
+      cffi_modules=['persistent/_ring_build.py:ffi'],
```

Comment:
Does this bit need a `setup_requires=["cffi>=1.0.0"]`?  That seems to be in the [example](https://cffi.readthedocs.io/en/latest/cdef.html#preparing-and-distributing-modules), anyway.

Reply:
No, that's not necessary. setuptools ignores unknown keyword arguments. 

It does mean that we need to make sure we have cffi installed before we build the wheels to distribute, though, if we want the cffi module to be built. This PR does that.